src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+@import "../utils/variables";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-size;
+  padding: 10px;
```

Comment:
i  think you don't need padding here, element is centerd and have space arount itself, padding is unnecessary here